ios/browser/api/ads/brave_ads.mm

```diff
@@ -567,14 +567,19 @@ - (void)reportMediaStoppedWithTabId:(NSInteger)tabId {
 }
 
 - (void)reportTabUpdated:(NSInteger)tabId
-                     url:(NSURL*)url
+      redirectedFromURLs:(NSArray<NSURL*>*)redirectionURLs
               isSelected:(BOOL)isSelected
                isPrivate:(BOOL)isPrivate {
   if (![self isAdsServiceRunning]) {
     return;
   }
-  ads->OnTabUpdated((int32_t)tabId, net::GURLWithNSURL(url), isSelected,
-                    [self isBrowserActive], isPrivate);
+  std::vector<GURL> urls;
+  for (NSURL* redirectURL in redirectionURLs) {
+    urls.push_back(net::GURLWithNSURL(redirectURL));
+  }
+  urls.push_back(net::GURLWithNSURL(url));
```

Comment:
You deleted the `url` argument, so this line has to be removed or it won't compile. If you want this API to mimic `reportLoadedPageWithURL` make sure to keep the `url` argument